test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_endpoint.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import pytest
+
+from sagemaker import utils
+from sagemaker.pytorch import PyTorchModel
+
+from ..... import invoke_sm_helper_function
+from ...integration import RESOURCE_PATH, model_cpu_dir, mnist_gpu_script
+from ...integration.sagemaker import timeout
+
+@pytest.mark.model("mnist")
+@pytest.mark.processor("gpu")
+@pytest.mark.gpu_test
```

Comment:
Do these tests add anything that existing functional tests don't have? If they are meant to just be triggered as smoke/sanity tests and not necessary to run in every RC pipeline, we can probably move these to their own directory under dlc_tests so that we don't add unneeded testing overhead in the RC creation process

Reply:
Markers will be added to selectively run these tests.